Makefile

```diff
@@ -137,17 +139,26 @@ LIMITADOR_OPERATOR_GITREF = $(LIMITADOR_OPERATOR_BUNDLE_VERSION)
 endif
 LIMITADOR_OPERATOR_BUNDLE_IMG ?= quay.io/kuadrant/limitador-operator-bundle:$(LIMITADOR_OPERATOR_BUNDLE_IMG_TAG)
 
-## policy-controller
-POLICY_CONTROLLER_VERSION ?= latest
-policy_controller_is_semantic := $(call is_semantic_version,$(POLICY_CONTROLLER_VERSION))
-ifeq (latest,$(POLICY_CONTROLLER_VERSION))
-POLICY_CONTROLLER_VERSION = 0.0.0
-POLICY_CONTROLLER_GITREF = main
-else ifeq (true,$(policy_controller_is_semantic))
-POLICY_CONTROLLER_GITREF = v$(POLICY_CONTROLLER_VERSION)
+## kuadrant dns
+#ToDo Pin this version once we have an initial release of kuadrant dns
+KUADRANT_DNS_OPERATOR_VERSION ?= latest
+KUADRANT_DNS_OPERATOR_VERSION = unified_kuadrant #ToDo(mnairn) Remove this before merge
+
+limitador_bundle_is_semantic := $(call is_semantic_version,$(KUADRANT_DNS_OPERATOR_VERSION))
```

Comment:
should this be `dns_bundle_is_semantic` ?

Reply:
yep, fixed now